website/docs/r/action_rule_profile.html.markdown

```diff
@@ -37,6 +43,9 @@ resource "aci_action_rule_profile" "example" {
 * `annotation` - (Optional) Annotation of the object Action Rule Profile.
 * `description` - (Optional) Description of the object Action Rule Profile.
 * `name_alias` - (Optional) Name alias of the object Action Rule Profile.
+* `set_communities` - (Optional) Set Communities of object Action Rule Profile.
```

Comment:
Document what are the expected attributes in the map and add type Map to the documentation.